proto/tpm_enrollz.proto

```diff
@@ -53,11 +53,18 @@ message RotateOIakCertRequest {
   ControlCardSelection control_card_selection = 1;
 
   // PEM-encoded owner initial attestation key certificate signed by the
-  // switch owner/administrator CA.
+  // switch owner/administrator CA. The PEM formatted cert string can have
+  // more than one certificate block representing a certificate chain.
```

Comment:
Should attestz also be updated?

https://github.com/openconfig/attestz/blob/560ec3522e3b5b892fabd1c72f3c97b620c0b632/proto/tpm_attestz.proto#L54-L57

Reply:
Done